### PR TITLE
Privacy not go brr haha

### PR DESCRIPTION
**Pull request by theodorsm**
he back

king_gdpr

### CODE REVIEW

**Review by LudvigHz — APPROVED** [review]
LGTM

**Comment by SmithPeder** [comment]
⛷️⛷️⛷️⛷️⛷️⛷️⛷️

**Comment by SmithPeder** [comment]
Let's get this bread

File: app/components/Footer/index.tsx [hunk 1]
```diff
@@ -129,6 +129,9 @@ const Footer = ({ loggedIn }: Props) => (
         <Link to="/pages/personvern/114-informasjonskapsler">
           Informasjonskapsler (cookies)
         </Link>
+        <Link to="/pages/personvern/124-personvernserklring">
+          Personvernserklæring
```
Comment (ivarnakken) [inline app/components/Footer/index.tsx:133]: personvernerklæring**

Comment (SmithPeder) [reply app/components/Footer/index.tsx:133]: image

Comment (theodorsm) [reply app/components/Footer/index.tsx:133]: image

image

Comment (theodorsm) [reply app/components/Footer/index.tsx:133]: image

Comment (theodorsm) [reply app/components/Footer/index.tsx:133]: Choose your fighter

**Pull request merged**